### PR TITLE
Intro duplicates

### PR DESCRIPTION
**Pull request by NielsZeilemaker**

### CODE REVIEW

File: community.py [hunk 1]
```diff
@@ -2189,39 +2204,76 @@ def on_identity(self, messages):
         """
         pass
 
-    def on_authorize(self, messages, initializing=False):
+    def create_signature_request(self, candidate, message, response_func, response_args=(), timeout=10.0, forward=True):
```
Comment (whirm) [inline community.py]: Could you wrap the lines to 120chars?

File: community.py [hunk 2]
```diff
@@ -2189,39 +2204,76 @@ def on_identity(self, messages):
...
-        @raise DropMessage: When unable to verify that this message is valid.
-        @todo: We should raise a DelayMessageByProof to ensure that we request the proof for this
-         message immediately.
-        """
-        for message in messages:
-            self.timeline.authorize(message.authentication.member, message.distribution.global_time, message.payload.permission_triplets, message)
+        Receiving the dispersy-signed-response message results in a call to RESPONSE_FUNC.  The
+        first parameter for this call is the SignatureRequestCache instance returned by
+        create_signature_request, the second parameter is the proposed message that was sent back,
+        the third parameter is a boolean indicating weather MESSAGE was modified.
```
Comment (whirm) [inline community.py]: whether

File: community.py [hunk 3]
```diff
@@ -2189,39 +2204,76 @@ def on_identity(self, messages):
...
 
-    def on_revoke(self, messages, initializing=False):
-        """
-        Process a dispersy-revoke message.
+        RESPONSE_FUNC must return a boolean value indicating weather the proposed message (the
```
Comment (whirm) [inline community.py]: idem

**Comment by NielsZeilemaker** [comment]
@whirm ready to be merged

**Pull request merged**